src/StackAdmin/Resources/Commands.Resources/Commands.Resources.Netcore.csproj

```diff
@@ -0,0 +1,81 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
```

Comment:
same comment, why do we need a netcore build?

Reply:
removed